go/src/meguca/common/posts.go

```diff
@@ -10,8 +10,9 @@ var ParseBody func([]byte, string, uint64, uint64, string, bool) ([]Link, []Comm
 // Board is defined to enable marshalling optimizations and sorting by sticky threads
 // easyjson:json //
 type Board struct {
-	Pages   int      `json:"pages"`
-	Threads []Thread `json:"threads"`
+	Pages   int           `json:"pages"`
+	Threads []Thread      `json:"threads"`
+	ModLog  []ModLogEntry `json:"modLog"`
```

Comment:
We may also be able to move the mod-log from the DB to the `Board` cache. Thoughts?

Reply:
Yes, we could, but I realized how many db calls that is per get post that I figured it would be waaaaaay more efficient to just embed the mod-log into the board struct, have it go into the cache, and then maybe move the mod-log totally from the DB to the cache.
Case in point: What I was doing before was really costly, and I think this may be more efficient.
This also gives up the option (I think) to just simply synchronize the board cache, or part of it, with the client through websockets. It may also mean reworking a few other things, but I think it'll come out more efficient.

Reply:
Correct me, if I'm wrong, but didn't we have boolean columns for this in the post table (banned, etc.)? These can be set during modetration action writes and the mod log entry can be retrieved, only when these are true. Note, that them being true does not mean an entry still exists.

Reply:
Yes, post contains `Banned`, `Deleted`, and now, `MeidoVision`, which are booleans.
You mean only if one of them are true, right?

Reply:
>and the mod log entry can be retrieved, only when these are true

Looking back at this, are you saying we should only retrieve the `ModLogEntry` if one of those are true?
Considering how websockets work, there's no point to that, since if it's being fired for delete/banned/meidovision, we already know that post is one of those. It's smarter to have one general mod-log fetch for the board in totality, to minimize db calls.
One db call for the entire mod-log for the board versus 10 calls to get 3 separate `ModLogEntry`s per-post at an exponential rate as users increase is a better idea.

Reply:
Also, again, by moving the mod-log to the cache per-board, we don't even have to call the db, thus freeing up the db a bit, and keeping board info, other than config, in one unified interface.
Also, I'm not entirely sure, but we also free up memory if the cache is filesystem-based, and not in memory.

Reply:
Hm, actually, looking at migrations.go again, maybe removing the mod-log for the cache is pointless, since posts themselves are, in the end, from the db.
I still think we should have the mod-log in the cache for `PageStore` (board) regardless, though, to reduce db calls to mod-log.

Reply:
Yeah, using the cache is pointless.
I can get the same result by just adding a websockets handler to sync the board mod-log with the client, and have the client fetch the mod-log locally from a struct by ID.

Reply:
Oh, and the irony is that I was doing more db calls.